test/controllers/tickets_controller_test.rb

```diff
@@ -320,6 +320,32 @@ class TicketsControllerTest < ActionController::TestCase
     end
   end
 
+  test 'shown captcha should not break new ticket page when not signed in' do
+    # Stub keys Recaptcha
+    Recaptcha.configuration.public_key = '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI'
+    Recaptcha.configuration.private_key = '6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe'
```

Comment:
These are loaded using your initializer and secrets file, you can remove these two lines.